package.json

```diff
@@ -1,141 +1,145 @@
-        "bazel:format-check": "yarn bazel:format --mode=check",
-        "bazel:lint": "yarn bazel:format --lint=warn",
-        "bazel:lint-fix": "yarn bazel:format --lint=fix",
-        "// See comment in .bazelrc": "",
-        "bazel:update-deleted-packages": "sed -i.bak \"/--deleted_packages/s#=.*#=$(find {examples,e2e}/*/* \\( -name BUILD -or -name BUILD.bazel \\) | xargs -n 1 dirname | paste -sd, -)#\" .bazelrc && rm .bazelrc.bak",
-        "update-codeowners": "./scripts/update_codeowners.sh",
-        "format": "git-clang-format",
-        "format-all": "clang-format --glob='{internal/**/,examples/**/}*.{js,ts}' -i",
-        "stardoc": "bazel build //docs && cp -f dist/bin/docs/*.md docs",
-        "version": "conventional-changelog -p angular -i CHANGELOG.md -s && node ./scripts/on-version.js && bazel build //:release && node ./scripts/on-release.js && git stage version.bzl docs/install.md packages/create/index.js README.md CHANGELOG.md e2e/*/WORKSPACE examples/*/WORKSPACE",
-        "postinstall": "patch-package && node internal/npm_install/test/postinstall.js"
-    },
-    "husky": {
-        "hooks": {
-            "pre-commit": "check-clang-format \"yarn format\"",
-            "pre-push": "yarn bazel:format-check && yarn bazel:lint",
-            "commit-msg": "commitlint -E HUSKY_GIT_PARAMS"
-        }
+  "name": "@bazel/javascript",
+  "description": "Build JavaScript with Bazel",
+  "version": "1.6.0",
+  "keywords": [
+    "javascript",
+    "bazel"
+  ],
+  "homepage": "https://github.com/bazelbuild/rules_nodejs",
+  "repository": "https://github.com/bazelbuild/rules_nodejs",
+  "license": "Apache-2.0",
+  "engines": {
+    "node": ">=12.0.0 < 13",
+    "yarn": ">=1.13.0"
+  },
+  "devDependencies": {
+    "@angular/common": "^9.1.0",
+    "@angular/compiler": "^9.1.0",
+    "@angular/compiler-cli": "^9.1.0",
+    "@angular/core": "^9.1.0",
+    "@babel/cli": "^7.6.2",
+    "@babel/core": "^7.6.2",
+    "@babel/preset-env": "^7.6.2",
+    "@bazel/bazelisk": "^1.5.0",
+    "@bazel/buildifier": "^3.2.0",
+    "@bazel/buildozer": "^3.2.0",
+    "@bazel/ibazel": "^0.13.1",
+    "@commitlint/cli": "^8.0.0",
+    "@commitlint/config-conventional": "^8.0.0",
+    "@cypress/browserify-preprocessor": "^3.0.0",
+    "@gregmagolan/test-a": "0.0.5",
+    "@types/express": "^4.17.6",
+    "@types/hammerjs": "2.0.35",
+    "@types/jasmine": "~3.3.13",
+    "@types/jest": "24.9.0",
+    "@types/node": "^12.0.0",
+    "@types/semver": "6.2.0",
+    "babel-jest": "^25.5.1",
+    "bazel_workspaces": "file:./tools/npm_packages/bazel_workspaces",
+    "bazel_workspaces_consistent": "file:./tools/npm_packages/bazel_workspaces_consistent",
+    "c8": "7.1.0",
+    "clang-format": "1.2.2",
+    "conventional-changelog-cli": "^2.0.21",
+    "core-util-is": "^1.0.2",
+    "cypress": "^4.7.0",
```

Comment:
I think you can revert this file? we don't require cypress anywhere in this workspace

Reply:
oh I see it's used in the test. let's move cypress depnedency to packages/cypress/package.json and install that as cypress_deps, like packages/angular does?

Reply:
done